tests/clients/test_instance_level.py

```diff
@@ -32,7 +32,7 @@ def __getitem__(self, index: int) -> Tuple[torch.Tensor, torch.Tensor]:
         return self.data[index], self.targets[index]
 
 
-class TestClient(InstanceLevelPrivacyClient):
+class ClientForTest(InstanceLevelPrivacyClient):
```

Comment:
These classes are just renamed to address a warning in pytest that it can't "discover" these tests. It just uses a rudimentary GREP for the prefix "Test" to find tests. So the rename just reduces the warnings.